server/neptune/gateway/event/deploy_workflow_signaler.go

```diff
@@ -79,6 +79,7 @@ func (d *DeployWorkflowSignaler) SignalWithStartWorkflow(
 				Credentials: workflows.AppCredentials{
 					InstallationToken: installationToken,
 				},
+				//TODO: Remove ref from revision arguments
```

Comment:
ref data is not provided in the checkrun suite, so I've hard coded a branch ref for now. I think we can get rid of it completely, but I wanted to confirm how important it is (only use case [here](https://github.com/lyft/atlantis/blob/release-v0.17.3-lyft.1/server/neptune/workflows/internal/terraform/job/cmd_step_runner.go#L34)).

Reply:
That's what I thought to, will remove.